LCM/dsc/engine/ConfigurationManager/LocalConfigManagerHelper.c

```diff
@@ -2075,31 +2164,51 @@ MI_Result ApplyPendingConfig(
         result = ClearBuiltinProvCache(PACKAGE_PROVIDER_CACHE, cimErrorDetails);
         if (result == MI_RESULT_OK)
         {
+            flags |= LCM_EXECUTE_APPLYNEWCONFIG;
```

Comment:
>         flags |= LCM_EXECUTE_APPLYNEWCONFIG; [](start = 3, length = 45)

Can you add comment about this flags update. There is not one widows either but good to add one nevertheless.

Reply:
Done